static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-support-baseline.js

```diff
@@ -0,0 +1,245 @@
+// @flow
+import {
+    makeABTest,
+    defaultPageCheck as isEpicCompatibleWithPage,
+} from 'common/modules/commercial/contributions-utilities';
+import { setupEpicInLiveblog } from 'common/modules/experiments/tests/acquisitions-epic-liveblog';
+import { viewsInPreviousDays } from 'common/modules/commercial/acquisitions-view-log';
+import {
+    submitInsertEvent,
+    submitViewEvent,
+} from 'common/modules/commercial/acquisitions-ophan';
+import mediator from 'lib/mediator';
+
+import config from 'lib/config';
+
+const baseCampaignCode = 'gdnwb_copts_memco_sandc_support_baseline';
+
+const makeSupportURL = (campaignCode: string): string =>
+    `https://support.theguardian.com/uk?INTCMP=${campaignCode}`;
+
+const makeBecomeSupporterURL = (campaignCode: string): string =>
```

Comment:
You can look at this function in contributions-utilities:

`
const addTrackingCodesToUrl = (base: string, campaignCode: string) => {
    const params = {
        REFPVID: (config.ophan && config.ophan.pageViewId) || 'not_found',
        INTCMP: campaignCode,
    };

    return `${base}?${constructURLQuery(params)}`;
};
`